src/index.html

```diff
@@ -7,8 +7,47 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header container">
+      <a href="index.html" class="logo">
+        <img src="./images/logo.png"
+        alt="LOGO"
+        class="logo__png">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="index.html" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a href="index.html" data-qa="hover"
+            class="nav__link">Laptops & Computers</a>
+          </li>
```

Comment:
Apply code styling recommendations from a checklist
```suggestion
          <li class="nav__item">
            <a 
              href="index.html" 
              data-qa="hover"
              class="nav__link"
            >
              Laptops & Computers
            </a>
          </li>
```